components/Player.js

```diff
@@ -1,11 +1,20 @@
 import propTypes from 'prop-types'
 import Card from './Card'
 import Image from 'next/image'
+import ReactTooltip from 'react-tooltip'
+import { useEffect, useState } from 'react'
 
 /**
  * Player list component
  */
 export default function Player(props) {
+  //Since we are using SSR, the below code is necessary to make sure the component is mounted before showing the tooltip
+  const [isMounted, setIsMounted] = useState(false)
+
+  useEffect(() => {
+    setIsMounted(true)
+  }, [])
+
```

Comment:
I don't believe this is necessary. Are you getting an error?

Reply:
This is required for ReactTooltip, else we get a console warning. See here for more info: https://github.com/DTS-STN/Scrum-Poker/pull/81#issuecomment-1084916624

Reply:
It's a bit of a hack, I'd recommend we use a different package next time for tool tips that is NextJS friendly

Reply:
I agree it's a bit hacky. This seemed like the best package at the time since it is by far the most popular, but I'd be open to exploring other options like [Popper](https://popper.js.org/docs/v2/tutorial/) or [Tippy.js](https://github.com/atomiks/tippyjs-react)